typescript/infra/scripts/deploy.ts

```diff
@@ -46,6 +48,13 @@ async function main() {
   const envConfig = getEnvironmentConfig(environment);
   const env = deployEnvToSdkEnv[environment];
 
+  const supportedChainNames =
+    environment === 'mainnet3'
+      ? mainnet3Chains
+      : environment === 'testnet4'
+      ? testnet4Chains
+      : undefined;
```

Comment:
I mainly added this to avoid an issue I was having related to basegoerli - but I probably overindexed on this instead of the actual solution which is probably the removal of basegoerli support